ui/src/app/settings/containers/Configuration/Configuration.js

```diff
@@ -1,5 +1,31 @@
-import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import PropTypes from "prop-types";
+import React, { useState, useEffect } from "react";
 
-const Configuration = () => <h4>Configuration</h4>;
+import actions from "app/settings/actions";
+import selectors from "app/settings/selectors";
+import baseSelectors from "app/base/selectors";
+import Loader from "app/base/components/Loader";
+
+const Configuration = () => {
+  const dispatch = useDispatch();
+  const configuration = useSelector(state =>
+    selectors.configuration.all(state)
+  );
+  const loading = useSelector(selectors.configuration.loading);
+  const loaded = useSelector(selectors.configuration.loaded);
+
+  dispatch(actions.configuration.fetch());
```

Comment:
This needs to be a [`useEffect`](https://reactjs.org/docs/hooks-effect.html), otherwise this component will fetch every time it renders. Have a look at the examples with `useFetchOnce`. That hook will be replaced soon (that logic belongs in a saga), but dispatching always needs to be part of the lifecycle or bound to an event.